src/token/HolographUtilityToken.sol

```diff
@@ -33,4 +33,11 @@ contract HolographUtilityToken is ERC20H {
     // run underlying initializer logic
     return _init(initPayload);
   }
+
+  /**
+   * @dev Temporarily placed to bypass bytecode conflicts
+   */
+  function isHLG() external pure returns (bool) {
```

Comment:
When will we remove this given the bycode conflict?

Reply:
This was temporarily added so that I could run extensive onchain tests on top of the regular localhost tests. Will be removed when we are ready to change deployment salt.